src/com/serotonin/mango/Common.java

```diff
@@ -26,12 +26,7 @@
 import java.security.NoSuchAlgorithmException;
 import java.text.MessageFormat;
 import java.text.ParseException;
-import java.util.ArrayList;
-import java.util.Enumeration;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Locale;
-import java.util.ResourceBundle;
+import java.util.*;
```

Comment:
Please keep previous imports with the explicit declaration.

Reply:
ok, thank you